snafu/utils/get_prometheus_data.py

```diff
@@ -13,29 +13,30 @@
 class get_prometheus_data():
     def __init__(self, action):
 
+        self.sample_info_dict = action
         self.uuid = action["uuid"]
         self.user = action["user"]
         self.cluster_name = action["cluster_name"]
         self.test_config = action["test_config"]
 
         # change datetime in seconds string to datetime object
-        starttime = datetime.fromtimestamp(int(action["starttime"]))
-        self.start = starttime.datetime()
+        starttime = datetime.fromtimestamp(int(self.sample_info_dict["starttime"]))
+        self.start = starttime
 
         # change datetime in seconds string to datetime object
-        endtime = datetime.fromtimestamp(int(action["endtime"]))
+        endtime = datetime.fromtimestamp(int(self.sample_info_dict["endtime"]))
         # add 120s buffer to end time
-        endtime = endtime + timedelta(seconds=120)
-        self.end = endtime.datetime
+        # endtime = endtime + timedelta(seconds=120)
+        self.end = endtime
 
         # step value to be used in prometheus query
-        self.T_Delta = 30
+        self.T_Delta = 15
```

Comment:
Do we want to allow the step value be overwritten via the commandline? default to 15, but allow the user to change it?

Reply:
The default scrape interval for OpenShift is 30s... but yeah, making this parameter configurable will make sense